package com.tpebank.exception.message;

public class ExceptionMessages {

    public final static String CONTACTMESSAGE_NOT_FOUND_MESSAGE="Contact message with id %d not found";
    public final static String USER_NOT_FOUND_MESSAGE="User with username %s not found";

    public final static String USERID_NOT_FOUND_MESSAGE="User with id %d not found";

    public final static String USERNAME_ALREADY_EXIST_MESSAGE="Username: %s already exist";

    public final static String EMAIL_ALREADY_EXIST_MESSAGE="Email: %s already exist";

    public final static String SSN_ALREADY_EXIST_MESSAGE="Ssn: %s already exist";

    public final static String ROLE_NOT_FOUND_MESSAGE="Role with role name %s not found";

    public final static String CURRENTUSER_NOT_FOUND_MESSAGE="Current user not found";

    public final static String ACCOUNT_NOT_FOUND_MESSAGE="Account not found with number %d";

    public final static String RECIPIENT_ADD_ERROR_MESSAGE="Recipient couldn't be added. Recipient account belongs this user!";

    public final static String RECIPIENT_ALREADY_EXIST_MESSAGE="This recipient has already saved for this user";

    public final static String RECIPIENT_VALIDATION_ERROR_MESSAGE="Recipient name has not matched";

}
